src/common/vars/static-files-directory.injectable.ts

```diff
@@ -4,16 +4,14 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import joinPathsInjectable from "../path/join-paths.injectable";
-import lensResourcesDirInjectable from "./lens-resources-dir.injectable";
 
 const staticFilesDirectoryInjectable = getInjectable({
   id: "static-files-directory",
 
   instantiate: (di) => {
     const joinPaths = di.inject(joinPathsInjectable);
-    const lensResourcesDir = di.inject(lensResourcesDirInjectable);
 
-    return joinPaths(lensResourcesDir, "static");
+    return joinPaths(__dirname, "..");
```

Comment:
Side-effect :)

Reply:
Right, you mean `__dirname` here? Just wondering what would be the proper injectable to get the same result.

Reply:
Fixed.